Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1691,6 +1691,11 @@ extension BrowserViewController: TabDelegate {
         updateFindInPageVisibility(visible: true)
         findInPageBar?.text = selection
     }
+
+    func tab(_ tab: Tab, didSelectSearchWithFirefoxForSelection selection: String) {
+        tab.isPrivate ? openBlankNewTab(focusLocationField: false, isPrivate: true) : openBlankNewTab(focusLocationField: false)
```

Comment:
I think this can just be `openBlankNewTab(focusLocationField: false, isPrivate: tab.isPrivate)`